test/basic_types/boolean.js

```diff
@@ -0,0 +1,15 @@
+'use strict';
+const buildType = process.config.target_defaults.default_configuration;
+const assert = require('assert');
+
+test(require(`../build/${buildType}/binding.node`));
+test(require(`../build/${buildType}/binding_noexcept.node`));
+
+function test(binding) {
+
```

Comment:
nit: Unnecessary blank line.

Reply:
Oh, okay I will remove it.
Thank you for review